benchmarks/inference-server/text-generation-inference/hpa-templates/hpa.cpu.yaml.tftpl

```diff
@@ -0,0 +1,13 @@
+apiVersion: autoscaling/v1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: tgi
+  namespace: ${namespace}
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: tgi
+  minReplicas: 1
+  maxReplicas: 5
```

Comment:
Please make the maxReplicas a variable in tfvars

Reply:
Done. I've also done the same for minReplicas.

For min, I've defaulted to 1 (which is reasonable). For max, I've defaulted to 5 (which is arbitrary - but at least controls costs if people blindly apply this.)